includes/class-wpcom-legacy-redirector-ui.php

```diff
@@ -125,10 +125,25 @@ public function custom_vip_legacy_redirect_column( $column, $post_id ) {
 				break;
 		}
 	}
+
+	/**
+	 * Get WP Home URL without path suffix
+	 *
+	 * @return string
+	 */
+	private function get_home_domain_without_path() {
```

Comment:
Unrelated change - fork from `develop` and apply the changes to that new branch.